Products/ZenUtils/controlplane/client.py

```diff
@@ -30,7 +30,18 @@
 
 
 SERVICED_VERSION_ENV = "SERVICED_VERSION"
-
+    
+def checkHothOrNewer():
```

Comment:
This method should return the cc version extracted from the env variable SERVICED_VERSION_ENV or "1.1.x" if the variable is not found